galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.mq.manager/src/main/java/dev/galasa/mq/IMessageQueue.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.mq;
+
+import javax.jms.Message;
+
+public interface IMessageQueue {
```

Comment:
@Michael-Baylis Adding you to this comment as I think this is something we should consider at a larger scale than just this manager.
But this manager will likely be implementing Pub and Sub at some point, and im wondering whether managers like this should have all the methods we plan inside an interface, that just throw unimplemented exceptions. Im only suggesting this as we are planning on doing this for a number of upcoming managers and it will allow users to understand that a managers capabilites are not limited, just not yet complete. This could also help us define the "completeness" of a manager once we start implementing all methos

Reply:
yep,  from next manager